pyproject.toml

```diff
@@ -104,6 +105,7 @@ plugins = [
     "numpy.typing.mypy_plugin",
 ]
 strict = true
+warn_return_any = false
```

Comment:
There are 17 errors which are ignored by this. They are almost all related to untyped external libraries. Do we want this here or the individual `# type: ignore[no-any-return]` comments?

Reply:
I will suggest adding individual # type: ignore[no-any-return]` comments to ensure that our functions don't violate this.

Reply:
Yeah makes sense, have done this.